distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -267,6 +274,8 @@ public void setup() throws Exception {
         server.start();
 
         cswServer.reset();
+        
+//        sleep(5000);
```

Comment:
Remove? Probably better to use `Object.wait()` instead of `Thread.sleep()` so we don't have to deal with the interrupted exception.